t/t-prune.sh

```diff
@@ -1066,6 +1066,7 @@ begin_test "prune doesn't hang on long lines in diff"
 
     git commit -m untracked
 
-    git lfs prune
+    GIT_TRACE=1 git lfs prune 2>&1 | tee prune.log
+    grep "Skipping long line ending with: A" prune.log
```

Comment:
Assuming my suggestion for the main long line handling code is OK, this will need to change to:
```shell
    grep "Ignoring trailing portion of long log line starting with: +AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA" prune.log
```